### ISSUE TITLE
"posted_on": "Failed to fetch!"

### ISSUE THREAD

**Issue by floriangeyer** [shaikhsajid1111/facebook_page_scraper#54]
I’m getting this error `"posted_on": "Failed to fetch!"` on specific posts. Any advice?

**Comment by shaikhsajid1111** [comment]
"Specific posts" not all posts?

**Comment by floriangeyer** [comment]
Right, they look identical from the HTML but it seems like either `parse(aria_label_value).isoformat()` or `Scraping_utilities._Scraping_utilities__convert_to_iso` fails on these specific posts. `aria_label_value` is fine for all posts, doesn’t include the exact time but its ok for me now.

**Comment by ihabpalamino** [comment]
hello guys if i want to fixe a period of date that i want to scrape how to do it?for example if i want to scrape from 2023/1/1 until 2023/4/13

**Comment by LUKE1860** [comment]
> I’m getting this error `"posted_on": "Failed to fetch!"` on specific posts. Other posts in the same feed are fine. Any advice?

I also got it, it occurs to me after 8 posts.
